cernopendata/modules/globals/ext.py

```diff
@@ -49,34 +51,47 @@ def __init__(self, app):
     def set_experiments(app):
         """Sets the experiments to be displayed in templates.
 
-        Use config `CERNOPENDATA_EXPERIMENTS` to exclude experiments.
-        For the experiment_data the following fields are currently supported:
-        name, url (of experiment), no_opendata_docs (exclude from about), height and width (image in footer)
+        Use environment variable `EXCLUDE_EXPERIMENTS` to exclude experiments.
```

Comment:
I'm concerned about the name of the variable. There is nothing CERNOPENDATA/INVENIO specific about it. What about renaming it to CERNOPENDATA_EXCLUDE_EXPERIMENTS?